docs/README.md

```diff
@@ -0,0 +1,103 @@
+# keanu-docs
+```
+    __ __                      
+   / //_/__  ____ _____  __  __
+  / ,< / _ \/ __ `/ __ \/ / / /
+ / /| /  __/ /_/ / / / / /_/ / 
+/_/ |_\___/\__,_/_/ /_/\__,_/  
+        documentation                  
+```
+by Improbable
+
+The source for the online documentation.
+
+[Internal documentation here](http://brevi.link/keanu)
+
+## Installation 
+These docs use [Jekyll](https://jekyllrb.com/docs/). Here are the steps you need to follow to get it up and running locally.
+
+If you don't have Ruby installed:
+* if you're using Windows, use [RubyInstaller](https://rubyinstaller.org/downloads/) to get Ruby with Devkit set up;
+* if you're using Mac, run `brew install ruby` from a terminal window.
+
+Install Jekyll.
+```bash
+gem install jekyll bundler
+```
+Move to _keanu-docs_ project directory and install project dependencies.
+```bash
+cd <keanu-docs_project_dir>
+bundle install
+```
+Optional: you may have issues with installing nokogiri. We solved our issues using the following command
+```bash
+gem install nokogiri -- --use-system-libraries --with-xml2-include=/usr/include/libxml2 --with-xml2-lib=/usr/lib
+```
+Now you can use our python script to generate the docs with snippets included...
+```bash
+python bin/snippet_writer.py
+```
+...and finally a local server that allows you to view the webpage at `localhost:4000`
+```bash
+bundle exec jekyll serve
+```
+
+## Compiling from a local build of Keanu
```

Comment:
Is this section now redundant that the `docs` will be part of `keanu`? So it'll always be using a local build?

Reply:
It most certainly is - The general walkthrough of using a local keanu build in other projects is also on the wiki so we can defo get rid of this.